styles/Card.module.css

```diff
@@ -0,0 +1,45 @@
+.card_list {
+  width: 530px;
+  display: grid;
+  grid-template-rows: auto auto;
+  gap: 16px;
+  border-right: 1px solid #191919;
+  padding-right: 48px;
+}
+
+.card_list__image {
+  background-color: #7d7d7d;
+  border-radius: 16px;
+  height: 100%;
+  object-fit: cover;
+  display: block;
+}
+
+.card_list:nth-last-child(1) {
+  border-right: none;
+  padding-right: 0;
+  margin-right: 0;
+}
+
+.card_list__text {
+  height: 107px;
+  display: grid;
+  grid-template-rows: auto auto;
+  row-gap: 8px;
```

Comment:
So many grids using on unnecessary places.

Reply:
Change back to flex